### PR TITLE
Dockerfile: use node:8-alpine as base image - Closes #2525

### LINKED ISSUE #2525: Change Node image used in Dockerfile to Alpine version
### Expected behavior

Use Alpine version in all our Docker third party images to minimise the size.

### Actual behavior

Postgres and Redis are pulling Alpine versions but Node doesn't.

### Which version(s) does this affect? (Environment, OS, etc...)

`1.2.0+`

### PR DESCRIPTION
**Pull request by ghost**
### How did I fix it?
Use `node:8-alpine` as base image; modify some debian-specific commands (user/group and package management)

### How to test it?
Build docker image and compare with the latest release on https://hub.docker.com/r/lisk/core/tags/; run the usual QA tests.

### Review checklist

* The PR resolves #2525 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -43,9 +41,6 @@ RUN if [ x"$( sha256sum /home/lisk/wait-for-it.sh |awk '{print $1}' )" = x"${WFI
       exit 1; \
     fi
 
-ENV LISK_API_WHITELIST=127.0.0.1
```
Comment (diego-G) [inline Dockerfile]: Why we don't need this anymore?

**Review by diego-G — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (fchavant) [reply Dockerfile]: It's in the defaults now: https://github.com/LiskHQ/lisk/blob/development/config/default/config.json#L35

**Review by 4miners — APPROVED** [review]

**Review by ManuGowda — APPROVED** [review]

File: docker/docker-compose.yml [hunk 2]
```diff
@@ -20,15 +20,15 @@ services:
       - LISK_DB_USER=${ENV_LISK_DB_USER}
       - LISK_DB_PASSWORD=${ENV_LISK_DB_PASSWORD}
       - LISK_DB_NAME=${ENV_LISK_DB_DATABASE}
+      - LISK_FILE_LOG_LEVEL=none
```
Comment (diego-G) [inline docker/docker-compose.yml]: Shall we get these vars from then environment?

**Review by diego-G — CHANGES_REQUESTED** [review]

File: docker/docker-compose.yml (hunk 2 above)
Comment (fchavant) [reply docker/docker-compose.yml]: Those are environment variables (defined here, will be set inside the container)

**Review by diego-G — APPROVED** [review]

**Review by MaciejBaj — APPROVED** [review]

**Pull request merged**